src/coreclr/clr.featuredefines.props

```diff
@@ -70,5 +70,11 @@
         <DefineConstants Condition="'$(TargetsUnix)' == 'true'">$(DefineConstants);TARGET_UNIX</DefineConstants>
         <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);TARGET_WINDOWS</DefineConstants>
         <DefineConstants Condition="'$(TargetsOSX)' == 'true'">$(DefineConstants);TARGET_OSX</DefineConstants>
+        <DefineConstants Condition="'$(TargetsLinux)' == 'true'">$(DefineConstants);TARGET_LINUX</DefineConstants>
```

Comment:
What about `mono`'s corelib?

Reply:
Would it be possible to move all `TARGET_XXX` constants to https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems, so that it gets naturally shared between CoreCLR and Mono?

Reply:
great catch, added!

Reply:
> Would it be possible to move all TARGET_XXX constants to https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems, so that it gets naturally shared between CoreCLR and Mono?

Thoughts on this?

Reply:
@jkotas done!

Reply:
We have I think at least TARGET_BROWSER defined also in few .csproj files. It'd be nice to clean them up as well.

Reply:
The convention so far has been to only define these constants locally in each .csproj file that needs them. It leads to some duplication, but it also avoids defining a ton of constants in each project that are not needed.